python/src/aiconfig/editor/server/server_utils.py

```diff
@@ -217,6 +219,9 @@ def init_server_state(app: Flask, edit_config: EditServerConfig) -> Result[None,
                 return Err(f"Failed to load AIConfig from {edit_config.aiconfig_path}: {e}")
     else:
         aiconfig_runtime = AIConfigRuntime.create()  # type: ignore
+        first_supported_model: str = ModelParserRegistry.parser_ids()[0]
```

Comment:
Is this guaranteed to have at least one element in it?

Reply:
I'm not sure -- do we support removing all parsers from the registry? By default they'd have all default models so someone would need to explicitly empty out the registry to hit this, right?

Reply:
Added a check to be safe